charts/k8ssandra/values.yaml

```diff
@@ -169,53 +198,148 @@ stargate:
-      username: ""
+    auth:
+      # -- Enables Stargate authentication ingress. Note: stargate.ingress.enabled must also be true.
+      enabled: true
+      # -- Optional hostname used to match requests, overriding stargate.ingress.host if set
+      host:
+      # tls:
+      #   # -- Hostnames to match as part of SNI routing
+      #   hosts:
+      #     - example.com
+      #   # -- Secret containing the TLS certificate and key for encrypting requests to this endpoint
+      #   secretName: my-secret
+
+    rest:
+      # -- Enables Stargate REST ingress. Note: stargate.ingress.enabled must also be true.
+      enabled: true
+      # -- Optional hostname used to match requests, overriding stargate.ingress.host if set
+      host:
+      # tls:
+      #   # -- Hostnames to match as part of SNI routing
+      #   hosts:
+      #     - example.com
+      #   # -- Secret containing the TLS certificate and key for encrypting requests to this endpoint
+      #   secretName: my-secret
+
+    graphql:
+      # -- Enables Stargate GraphQL API ingress. Note: stargate.ingress.enabled must also be true.
+      enabled: true
+      # -- Optional hostname used to match requests, overriding stargate.ingress.host if set
+      host:
+      # tls:
+      #   # -- Hostnames to match as part of SNI routing
+      #   hosts:
+      #     - example.com
+      #   # -- Secret containing the TLS certificate and key for encrypting requests to this endpoint
+      #   secretName: my-secret
+      playground:
+        # -- Enables GraphQL playground ingress.  Note: stargate.ingress.enabled and stargate.ingress.graphql.enabled must also be true.
+        enabled: true
+
+    cassandra:
+      # -- Enables C* native protocol ingress with Traefik. Note that this is mutually exclusive with cassandra.ingress.enabled, and stargate.ingress.enabled must also be true.
+      enabled: true
+      # -- Determines which TCP-based ingress custom resources to template out. Currently only `traefik` is supported
+      method: traefik
+
+      # -- Optional hostname used to match requests. Warning: many native Cassandra clients, notably including cqlsh, initialize their connection by querying for
+      # the cluster's contactPoints, and thereafter communicate to the cluster using those names/IPs rather than whatever host was specified to the client. In
+      # order for clients to work correctly through ingress with a host filter, this means that the host filter must match the hostnames specified in the
+      # contactPoints. This value must be a DNS-resolvable hostname and not an IP address. To avoid this issue, leave this setting blank, or if
+      # stargate.ingress.host is set, override it here to "" (empty string). This note does not apply to clients of Stargate's auth, REST, or GraphQL APIs.
+      host:
+
+      # -- Parameters used by the Traefik IngressRoute custom resource
+      traefik:
+
+        # -- Traefik entrypoint where traffic is sourced.
```

Comment:
This comment is referencing a property from the traefik chart installation. We should reference the docs to make it clear where it is coming from.

Reply:
Done.